.changelog/40186.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resources/aws_sagemaker_domain: Add `hidden_sagemaker_image_version_aliases` argument
```

Comment:
change to default_user_settings.studio_web_portal_settings. hidden_sagemaker_image_version_aliases